### ISSUE TITLE
URL update to match organization rename to NEONScience

### ISSUE THREAD

**Issue by mjones01** [NEONScience/NEONgitTraining#1]
Change NEONInc to NEONScience needed: 
* URL at top of the GitHub Code page
*  URL in the Config file.  

I'm also trying to figure out who, if anyone, monitors/maintains this repo.

**Comment by scelmendorf** [comment]
@mjones01 I don't think anyone maintains it.  We used it for in-house training when we were getting all the science staff running on git.  I think most people are there now.  You're welcome to recycle it into edu materials or kill it.

**Comment by mjones01** [comment]
@scelmendorf Thanks for the info.  I don't have permissions to do anything with this repo -- nor can I see who does have permissions.  With Leah the only one to have committed to the repo, I wasn't sure if this was something else I was supposed to manage as part of the NEON Data Skills programs.
As this is a public repo I think we should clean it up (delete or at least update the links to work) prior to pointing people to the NEONScience git hub organization for code resources. 

NDS.org also has a Git Training section we use with the Data Institute participants and it could be used with NEON staff (http://neondataskills.org/tutorial-series/GitHub/ ; but I just realized the tutorials are no longer showing on this page... I need to figure out why not).  However, it does focus on a forking workflow which the NEON Science staff that I've talked with don't use.

**Comment by cklunch** [comment]
@mjones01 @scelmendorf I do still direct new hires to the webpage made out of this repo, so I think it's still helpful. We could turn it to private, though. Sarah is an admin on the repo, I'm not.

**Comment by scelmendorf** [comment]
Sure - do you want me to make it private?

**Comment by mjones01** [comment]
@cklunch I'm guessing I should therefore make a PR with the needed changes to the URL so the page correctly builds?   The other options would just be to send them to the SWC lessons that this page appears to be built from:  http://swcarpentry.github.io/git-novice/

**Comment by cklunch** [comment]
Yeah, let's make it private. Unless it has to be public to make the webpage? I have no idea about that.

@mjones01 If you're up for doing a PR with the changes, that would be great! Thanks!

**Comment by mjones01** [comment]
@scelmendorf  PR submitted.  However, you need to go into the settings and change the URL for the page to be shown on from neoninc.github.org to neonscience.github.org.  

I also changed some settings in the config file to try to get the page to render correctly on my fork.  I couldn't get it to work but this may be because it was a different URL in the config file.  I'm happy to trouble shoot the site more but want to understand: 

1) That there is enough traffic/use of the site to make it worthwhile (and that NEON needs this and not just sending new employees to NDS or SWC's lessons on Git/GitHub). 
2) Where to charge trouble shooting time as I don't have anything for internal training materials.

**Comment by scelmendorf** [comment]
I merged the pull request.
Claire was already an administrator.
Megan is one now too.
Shall we make it private?

**Comment by mjones01** [comment]
@scelmendorf @cklunch - Hi.  I realized that even though we discussed making this a private repo in March we didn't do it.  Following up on a conversation with Wendy it looks like we're going to start advertising the NEONScience Git organization.  Should I go ahead and make this private at this time?  

Also, I'd be happy to restructure this repo to a simple training repo (without all the website files) and then direct users to the existing NEON Data Skills Git/GitHub lessons.  It teaching a forking workflow that not all (none ?) of the science teams use, but I'd be happy to work with someone to modify the current tutorials to be tailored for internal staff.

**Comment by scelmendorf** [comment]
Sure - feel free to make it private.  S
